.github/workflows/label.yml

```diff
@@ -0,0 +1,22 @@
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
```

Comment:
I think this part of the description is misleading because we're actually applying the label based on the target branch name (not the paths modified in the PR).

Reply:
Yeah that was part of the template, didn't bother but can remove